docs/USAGE.md

```diff
@@ -0,0 +1,114 @@
+# General Usage
+  
+## Packer Template:
+
+In order to use the orka packer plugin, you will have to define a packer template file.  The packer template file is the configuration   
+file that is used to define how to build an orka image.  You can either use JSON or HCL to create the template. Because Packer recommends  
```

Comment:
```suggestion
file that is used to define how to build an Orka image.  You can either use JSON or HCL to create the template. Because Packer recommends  
```